#VRML_SIM R2022b utf8

EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/backgrounds/protos/TexturedBackground.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/floors/protos/RectangleArena.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/robots/gctronic/e-puck/protos/E-puck.proto"

WorldInfo {
}
Viewpoint {
  orientation -0.5773502691896257 0.5773502691896257 0.5773502691896257 2.0944
  position -7.569471858517871e-06 -7.5694932621836355e-06 7.32303369680341
}
TexturedBackground {
}
TexturedBackgroundLight {
}
RectangleArena {
  floorSize 3 3
}
DEF epuck E-puck {
  translation -6.837564977904598e-11 -2.514257498543094e-16 -2.55177461898487e-05
  rotation 4.6626655925182025e-05 4.662682737765374e-05 0.999999997825947 1.5708000060964018
  controller "task1Solution"
  supervisor TRUE
  turretSlot [
    DEF Front DistanceSensor {
      translation 0.045 0 -0.011
      name "front distance sensor"
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    DEF Right DistanceSensor {
      translation 0.01 -0.035 -0.011
      rotation 0 0 1 -1.5707853071795865
      name "right distance sensor"
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    DEF Rear DistanceSensor {
      translation -0.025 0 -0.011
      rotation 0 0 1 3.1415
      name "rear distance sensor"
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    DEF Left DistanceSensor {
      translation 0.01 0.035 -0.011
      rotation 0 0 1 1.5708
      name "left distance sensor"
      recognitionColors [
        0 0 0
      ]
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    InertialUnit {
      rotation -1 0 0 -5.307179586466759e-06
    }
    Camera {
      translation 0.043 0 -0.015
      name "camera1"
      recognitionColors [
        1 1 0
      ]
      fieldOfView 1.57
      width 80
      height 80
      recognition Recognition {
      }
    }
  ]
}
